.github/ISSUE_TEMPLATE/bug-report.yml

```diff
@@ -1,6 +1,5 @@
 name: Bug Report
 description: Report a bug or regression in functionality
-labels: [bug]
```

Comment:
The "bug" label will not be added by default anymore. I deleted this because many opened bug reports are questions.